packages/e2e/e2e_macos/analysis_options.yaml

```diff
@@ -0,0 +1,11 @@
+# This is a temporary file to allow us to land a new set of linter rules in a
+# series of manageable patches instead of one gigantic PR. It disables some of
+# the new lints that are already failing on this plugin, for this plugin. It
+# should be deleted and the failing lints addressed as soon as possible.
```

Comment:
Not a blocker, but why do we need this for a new package? Shouldn't we just fix anything caught by these checks in the initial landing?

In fact, as a federated implementation is there even any Dart to lint?

Reply:
I was getting errors when the CI was running pub get on e2e_macos. It is possible we don't need this, so I'll revisit once I add more plugins